src/components/Progressbar/Progressbar.tsx

```diff
@@ -0,0 +1,75 @@
+import { Progress } from '@chakra-ui/react'
+import { Text } from '@chakra-ui/react'
+import style from './Progressbar.module.css'
+
+
+interface Props {
+    /**
+     * Цвет прогресс бара
+    */
+    colorScheme?: string,
```

Comment:
У прогресса в chakra UI есть список цветов. Только его и будем принимать
```suggestion
    colorScheme?: "blue" | "whiteAlpha" | "blackAlpha" | "gray" | "red" | "orange" | "yellow",
```